imapd.adoc

```diff
@@ -0,0 +1,19 @@
+= IMAPD Service
+
+
+Version {product-version} of {product-name} introduced a new
+`zimbra-imapd` service that allows you to run the IMAP[S]
+endpoints outside of the `mailboxd` process. This new service may be
+deployed on your mailbox servers or on stand-alone machines and it
+gives you the ability to scale IMAP separately from
+the mailboxes. This will also reduce the load on your mailbox servers.
+
+Although installation of the new `zimbra-imapd` is intended for larger
+multi-server installations, it is also fine to use with a
+single-server installation.
```

Comment:
I'd recommend some more text here.  Perhaps:
Using `zimbra-imapd` on a single server means that 2 different JVMs will be used to service IMAP requests instead of one, which means that each of them will use less memory which may improve garbage collection performance in each JVM.  The separation will improve the robustness of the system.  However, this needs to be balanced against the additional overhead of communication between the JVMs.